examples/config/config

```diff
@@ -516,7 +517,7 @@ set ssl_policy fail
 
 # === Post-load misc commands ================================================
 spawn_sync_exec @scripts_dir/load_cookies.sh
-spawn_sync_exec @scripts_dir/load_cookies.sh @data_home/session-cookies.txt
+spawn_sync_exec @scripts_dir/load_cookies.sh @(echo "${UZBL_SESSION_COOKIE_FILE}")@
```

Comment:
Hrm. This doesn't work if this isn't set in the environment :( . Maybe a variable should be used that cookies.py sets? Alternatively, pass a list of paths to `load_cookies.sh` and load the first (all?) of them.